IndexerGUI/IndexerGUI.csproj

```diff
@@ -226,6 +226,9 @@
   </ItemGroup>
   <ItemGroup>
     <Resource Include="Icons\Indexer++Logo.png" />
+    <EmbeddedResource Include="defaultUserSettings.xml">

```

Comment:
Sorry, my fault that I didn't create the CONTRIBUTING file with a reference to code style guidelines we use. Have already fixed that. For resources name, we should use [upper case](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-resources).